app/uk/gov/hmrc/cdsreimbursementclaimfrontend/controllers/claims/SelectDutiesController.scala

```diff
@@ -81,7 +82,7 @@ class SelectDutiesController @Inject() (
 
   def selectDutiesSubmit(): Action[AnyContent] =
     authenticatedActionWithSessionData.async { implicit request =>
-      withAnswers[DutiesSelectedAnswer] { (fillingOutClaim, _) =>
+      withAnswersAndRoutes[DutiesSelectedAnswer] { (fillingOutClaim, _, router) =>
```

Comment:
did you change your mind about altering the route file entries to have hardcoded values that are passed into the controller method?

Reply:
can you show the definition of `withAnswersAndRoutes`?